dotcom-rendering/src/client/islands/initHydration.ts

```diff
@@ -36,14 +37,15 @@ export const initHydration = async (
 ): Promise<void> => {
 	const name = getName(element);
 	const props = getProps(element);
+	const config = getConfig(element);
```

Comment:
related to @mxdvl's question about if config will be the same for every island (which I think it will be), I wonder if we need to `getConfig` from this element, or if we could store page config once for the page, and each island picks it up from there instead of every island having an identical `config` attribute?